### DISCUSSION TITLE
Reset

### DISCUSSION THREAD

**Discussion by mattyg2301** [postalserver/postal#2835] [Help with using Postal]
Hi,

Is there a way to flush all the queues etc, I had an issue which has resulted in the message held queue just growing, it's old messages that had an issue which got resolved after an upgrade but they now just keep on growing the queue.

The server is working fine and all mails that have been sent in the last week have been delivered. I'm just wanting to reset everything so that the stats below start from scratch again if this makes sense.

[25338 messages held]
[2314 queued messages]
[104.8% bounce rate]
[4.21 GB used]

**Comment by willpower232** [comment]
You could reduce the `Message retention days`, `Raw message retention days` and `Raw message retention size` settings under the advanced settings as a shortcut to reducing them but other than that, I'm not sure there is a way.
